Dockerfile

```diff
@@ -59,10 +57,12 @@ ENV RAILS_ENV=production \
 
 WORKDIR $APP_HOME
 
-RUN rm -rf $BUNDLE_PATH/$GEM_PATH/cache/*.gem && \
-    find $BUNDLE_PATH/$GEM_PATH/gems -name "*.c" -delete && \
-    find $BUNDLE_PATH/$GEM_PATH/gems -name "*.h" -delete && \
-    find $BUNDLE_PATH/$GEM_PATH/gems -name "*.o" -delete
+RUN rm -rf /usr/local/bundle/cache/*.gem && \
+    find /usr/local/bundle/gems -name "*.c" -delete && \
+    find /usr/local/bundle/gems -name "*.h" -delete && \
+    find /usr/local/bundle/gems -name "*.o" -delete && \
+    find /usr/local/bundle/gems -name "*.md" -delete && \
+    find /usr/local/bundle/gems -name "*.html" -delete
```

Comment:
This makes me a bit anxious as gems could conceivably use `html` files for providing functionality.

Reply:
ah okie, size gain its worth the risk then, I'll not delete the html and md files then.

Reply:
I could see only about 4MB gain by removing html files in the gems folder, have removed this line.